db/models.py

```diff
@@ -1,4 +1,11 @@
 from django.db import models
+from django.db.models import UniqueConstraint
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
+
+
+class User(AbstractUser):
```

Comment:
The `User` model extends `AbstractUser`, which is a good practice for customizing the user model. Ensure that this custom user model is correctly integrated with Django's authentication system.